data_structures/binary_tree/binarytree.go

```diff
@@ -1,13 +1,6 @@
 // Package binarytree basic binary tree and related operations
 package binarytree
 
-// Max Function that returns max of two numbers - possibly already declared.
-func Max(a, b int) int {
```

Comment:
Why remove it here and add to btree?

Reply:
If you look at the directory of the binary tree, you will see there are 4 files "binarysearchtree", "binarytree", "btree" and node. 
between those files "binarysearchtree" and "binarytree" just had a main function with bunch of test command, inside the "binarytree" we just had the Max function that it was used just in "btree" file so, I thought to be more clear I should create testcase but at that time I just focused on the structure so I just move the Max function into btree file to later make a test case file for it and remove those files "binarysearchtree" and "binarytree"